data.py

```diff
@@ -0,0 +1,168 @@
+# Will update the graphQL db by calling scraper.py every defined time interval
+from datetime import datetime
+import requests
+import schema
+from bs4 import BeautifulSoup
+
+URL = 'https://now.dining.cornell.edu/api/1.0/dining/eateries.json'
+PAYMENT_METHODS = {
+  'swipes': 'Meal Plan - Swipe',
+  'brbs': 'Meal Plan - Debit',
+  'cornell_card': 'Cornell Card',
+  'credit': 'Major Credit Cards',
+  'mobile': 'Mobile Payments'
+}
+
+eateries = {}
+operating_hours = {}
+events = {}
+menus = {}
+items = {}
+
+def start_update():
+  try:
+    print('[{0}] Updating data'.format(datetime.now()))
+    result = requests.get(URL)
+    data_json = result.json()
+    parse(data_json)
+    data = schema.Data()
+    data.update_data(
```

Comment:
`Data` should be a static singleton class and not actually instantiated, otherwise you'll get a new object every update